radiation/python/radsw/spcvrtm_gt4py.py

```diff
@@ -0,0 +1,1784 @@
+import sys
+import numpy as np
+import xarray as xr
+import time
+from gt4py.gtscript import (
+    BACKWARD,
+    FORWARD,
+    stencil,
+    function,
+    computation,
+    PARALLEL,
+    interval,
+    abs,
+    max,
+    min,
+    sqrt,
+)
+
+# sys.path.insert(0, "/Users/AndrewP/Documents/work/physics_standalone/radiation/python")
+sys.path.insert(0, "/work/radiation/python")
+from config import *
+from radsw.radsw_param import (
+    ngptsw,
+    nblow,
+    NGB,
+    idxsfc,
+    ntbmx,
+    bpade,
+    oneminus,
+    ftiny,
+    flimit,
+    eps,
+    nuvb,
+)
+from radphysparam import iswmode
+from util import create_storage_from_array, create_storage_zeros, compare_data
+
+# SERIALBOX_DIR = "/Users/AndrewP/Documents/code/serialbox2/install"
+SERIALBOX_DIR = "/usr/local/serialbox"
+sys.path.append(SERIALBOX_DIR + "/python")
+import serialbox as ser
+
+ddir = "/work/radiation/fortran/radsw/dump"
+# ddir = "/Users/AndrewP/Documents/work/physics_standalone/radiation/fortran/radsw/dump"
+serializer = ser.Serializer(ser.OpenModeKind.Read, ddir, "Serialized_rank1")
```

Comment:
only rank 1?

Reply:
Yes I've just been working on one rank so far. I can move to more ranks next